x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```diff
@@ -143,12 +143,33 @@
           }
         },
         {
-          "ecs_usage_scaled_float": {
+          "ecs_usage_double_scaled_float": {
             "mapping": {
               "type": "scaled_float",
               "scaling_factor": 1000
             },
-            "path_match": "*.usage"
+            "path_match": "*.usage",
+            "match_mapping_type": "double"
```

Comment:
It's unfortunate that this change makes the templates quite a bit more complex. Will https://github.com/elastic/elasticsearch/issues/102795 bring this back to just one entry? Alternative would be to support `match_mapping_type` as array like we did for other configs.

Reply:
This PR adds both `unmatch_mapping_type` and the ability to have arrays: https://github.com/elastic/elasticsearch/pull/103171. With this, we'll get back to one entry. We should still move forward with this PR and back port it to 8.12.